workspace/dao.py

```diff
@@ -0,0 +1,73 @@
+from db_classes import Workspace
```

Comment:
I'm very likely missing something but how are these functions guaranteed a connection to the database? Is this somewhere where we could use the helper functions from AGENDUM-57?